### PR TITLE
Added rename file feature as a button

### LINKED ISSUE #88: Documentation - Add rename file feature
Users should be able to rename their files in case they make a mistake when first naming it.

### PR DESCRIPTION
**Pull request by cheung-arthur**
Fixes #88 

Made a class for "Create New file" and "Rename file" for styling purposes

Added functionality for renaming a file

### CODE REVIEW

**Review by nicholas-ngyn — APPROVED** [review]
Nice idea! This feature enables the user to edit the file name which I overlooked to add during the first time.

File: src/scripts/main/public/js/documentation.js
Comment (kevku) [inline src/scripts/main/public/js/documentation.js:1]: What should the function do when the file name is empty?

**Review by kevku — APPROVED** [review]
Checks for cases when its empty or when you try to use a name that already exits, then edits file name

File: src/scripts/main/public/js/documentation.js
Comment (cheung-arthur) [reply src/scripts/main/public/js/documentation.js:1]: It will keep the previous name for the file - (i.e the renaming doesn't happen)

**Pull request merged**